en/docs/get-started/get-started.md

```diff
@@ -0,0 +1,470 @@
+
+To consume the REST API, the `readingListApp` application needs to subscribe to it. To subscribe your application to the API, follow the steps given below:
+
+1. In the left navigation menu, click **Subscriptions**.
+
+2. Click **Add APIs**.
+
+3. Find your REST API and click **Add**.
+
+Now your application has subscribed to the `readingList` REST API.
+
+
+### Step 2.3: Deploy a Web application and invoke the REST API
+
+At present, any user can invoke the `readingList` REST API via the `readingListApp` application (i.e., using its token) and update the same reading list. To allow multiple users to use the application and maintain unique reading lists, you need a front-end application that allows each user to log in with a unique user ID.
+
+In this step, let's deploy a pre-designed front-end application. This application is designed to personalize the readingList based on the user ID that it obtains from its identity provider. To enable the application to obtain the user ID, let's configure Asgardeo as the identity provider.
+
+
+#### Step 2.3.1: Create a single-page application in Asgardeo
+
+To generate the configurations required for end users to log in to the front-end application (for example, the access token, redirect URLs, etc.), let's create an application in Asgardeo by following the steps given below:
+
+1. Access Asgardeo at [https://console.asgardeo.io/](https://console.asgardeo.io/) and log in with the same credentials with which you logged in to Choreo.
+
+2. Click **Single-Page Application** to start adding a single-page application.
+
+3. Enter a name for the single-page application and enter `https://localhost:5173` as the authorized redirect URL.
+
+4. Click **Register**.
+
+5. Click **Protocol** to open the **Protocol** tab, and make the following changes:
+
+    1. In the **Authorized redirect URLs** field, enter `http://localhost:5173/`. Next, click the **Add** icon.
+   
+    2. Under **Access Token**, select **JWT** as the token type.
+   
+    3. Scroll down to the **ID Token** section and enter the following value in the **Audience** field.
+
+        `https://sts.choreo.dev/oauth2/token`
+
+         Click **Add URL** to save this value.
+   
+    4. Click **Update**.
+
+7. Click **Quick Start**, and then click the **React** icon (because you will be using a pre-configured sample react front-end application).
+
+    A separate page opens for your application with **Integrate Your Application** option selected by default.
+
+    !!! info   
+        Under this option, you will see some instructions to configure the application. Some of these steps are already completed for the pre-configured front-end application you will be using.
+        
+Next, you need to pass the configurations displayed on this page to your front-end application. To do so, proceed to [Step 2.4.2 - Configure the front-end application](#step-242-configure-the-front-end-application).
+
+### Step 2.3.2: Configure the front-end application
```

Comment:
Configure the front-end application to integrate with Asgardeo

Reply:
Integration with Asgardeo is not the only thing we are covering in this section.

Reply:
What else are we doing in this section? 
The heading level is not consistent with its same level.

Reply:
Changed the header style.

Here, we also give the cosumer key so that the application can access the REST API.